c7n/resources/ami.py

```diff
@@ -278,6 +278,44 @@ class ImageAgeFilter(AgeFilter):
         days={'type': 'number', 'minimum': 0})
 
 
+# NOTE should refactor a common filter to support all attributes
+@AMI.filter_registry.register('last-launched-time')
+class ImageLastLaunchedTimeFilter(AgeFilter):
```

Comment:
fwiw, in general, age filter base is a legacy thing, given value filter has support for it and wider range of conversions and acceptable formats.

Reply:
it does feel like this could be made a generic image attribute filter by reusing value filter.

Reply:
Just to clarify, I am not working on refactor the filter for this PR. So to whom it may concert, this conversation is not going to be resolved (not by me at least).